challenges/objects-arrays.js

```diff
@@ -80,8 +119,10 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 let lowerCase = [];
+zooAnimals.map(animal_name => {return animal_name.toLowerCase();});
```

Comment:
Your function should not also be called animal_name because that is the name of one of your keys. Name it like animal and then access animal_name through the function like so: animal => { return animal.animal_name.toLowerCase();